### PR TITLE
Modified tests June 17

### PR DESCRIPTION
**Pull request by manjunath-grl**
Summary:Modified tests
TC-BOOl-1.1
TC-CC-1.1
TC-FLW-1.1
TC-I-1.1
TC-ILL-1.1
TC-ILL-2.1
TC-LVL-1.1 to TC-LVL-6.1
TC-MC-1.*
TC-MOD-1.1
TC-OCC-1.1
TC-OCC-2.1
TC-OO-1.1
TC-OO-2.4
TC-PCC-1.1
TC-PRS-1.1
TC-PS-1.1
TC-RH-2.1
TC-TSUIC-1.1
TC-ULABEL-1.1
TC-WNCV-1.1
TC-WNCV-2.1
TC-WNCV-4.3
Added auto generated files

Attached execution logs for reference:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/8924185/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml [hunk 1]
```diff
@@ -50,13 +50,12 @@ tests:
               minValue: 0
               maxValue: 32768
 
-    #Issue 18041
     - label: "TH reads from the DUT the (0xFFFB) AttributeList attribute"
       PICS: PICS_SKIP_SAMPLE_APP
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:54]:
```suggestion
      PICS: PICS_SKIP_SAMPLE_APP
```
are we sure about this pics ?

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml [hunk 2]
```diff
@@ -50,13 +50,12 @@ tests:
...
       command: "readAttribute"
       attribute: "AttributeList"
       response:
-          value: [0, 7, 10, 13, 23, 26, 65528, 65529, 65531, 65532, 65533]
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:59]: what was 26 ?

Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:59]: @jmeg-sfy its an optional attribute (26) SafetyStatus (O) we are just checking for Mandatory attributes for now.
image

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:54]: Hi @jmeg-sfy @raju-apple ya we added this PICS just to skip steps that are currently not supported or do not run on the CI, If this [PR](https://github.com/project-chip/connectedhomeip/pull/19730) merges we will remove this PICS condition

Comment (jmeg-sfy) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:54]: ok good to know thanks

**Comment by woody-apple** [comment]
Fast tracking tests.

**Pull request merged**